editor/inspector/assets/fbx/animation.js

```diff
@@ -303,14 +303,14 @@ exports.$ = {
 const Elements = {
     // infos put first
     infos: {
-        ready() {
+        ready () {
```

Comment:
Did you  need Spaces after function names?
If necessary, please format all the code correctly in the standalone PR.

Reply:
yes
latter